discover/portal/articles/210-setting-up/01-system-settings/01-system-settings.markdown

```diff
@@ -0,0 +1,121 @@
+# System Settings [](id=system-settings)
+
+@product@ is modular, meaning it's composed of many applications, which are
+divided into even smaller "chunks" of functionality. The applications, and
+sometimes even code chunks, are configurable at several scopes, as discussed in
+the introductory article for this section. 
+
+To make configuration changes to an application in a system-wide fashion, use
+the System Settings application at Control Panel &rarr; Configuration &rarr;
+System Settings.
+
+![Figure 1: System Settings are accessed through the Control Panel.](../../../images/system-settings-categories.png)
+
+## Editing the Default Configuration [](id=editing-the-default-configuration)
+
+In System Settings, the configuration options are organized into logical
+sections (for example, Content) and categories (for example, Workflow) based on
+their functionality. There's also a Search bar to make finding configuration
+entries easier. 
+
+![Figure 2: System Settings are organized by component.](../../../images/system-settings-nav-search.png)
+
+Changing an application's default configuration options in System Settings isn't
+difficult. Once you find what you're looking for, simply click the name of the
+component you want to configure, or click the actions button
+(![Actions](../../../images/icon-actions.png)) then click *Edit*. Make any
+changes you'd like, then click *Save*. Your configuration changes are saved and
+applied throughout the system. 
+
+![Figure 3: After saving changes to a configuration, the actions *Reset Default Values* and *Export* are available for it.](../../../images/system-settings-actions.png)
+
+Keep in mind that some configurations are available at more than one scope. Once a
```

Comment:
I would be a bit more specific. As mentioned above, when the configuration is of a scope different than system, what is being done within System Settings is to set the default.

Reply:
That's why I added the example about Blogs Entries Comments below, to demonstrate how this works. It's not enough?